admin/app/views/workarea/admin/users/edit.html.haml

```diff
@@ -36,6 +36,10 @@
           = label_tag 'user[password]', t('workarea.admin.fields.password'), class: 'property__name'
           = password_field_tag 'user[password]', nil, class: 'text-box', autocomplete: 'off'
           %span.property__note= link_to t('workarea.admin.users.edit.send_password_reset'), send_password_reset_user_path(@user), data: { method: :post }
+        .property
```

Comment:
This is a less-like-to-be-used field, can you add it to the grid with the email signup toggle?